pycookiecheat/pycookiecheat.py

```diff
@@ -124,7 +138,10 @@ def get_linux_config() -> dict:
     return config
 
 
-def chrome_cookies(url: str, cookie_file: str = None) -> dict:
+def chrome_cookies(
+        url: str,
+        cookie_file: str = None,
+        browser_name: str = None) -> dict:
```

Comment:
Actually, you could just set the default browser to `Chrome` here instead of duplicating the work above -- I don't think either of those config utility functions will be reused outside of this library, so probably not a lot of benefit in giving `browser` a default there if it's only going to be called from here.